### PR TITLE
Add space templates

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Description
Eases the creation of spaces, so no readme must be created, since we use the server space template

This pr also fixes various bugs:
* create/ edit Space description (readme) not possible when readme.md file was missing in .space folder
* .space folder not listed in the resource table, when created via image or icon upload (when .space folder was missing)
* space icon not present in the resource table, when no image was set 

## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by AlexAndBear** [comment]
blocked by https://github.com/owncloud/ocis/issues/8713#issuecomment-2012250172

**Comment by AlexAndBear** [comment]
I will leave this PR open until https://github.com/owncloud/ocis/issues/8719 gets done, so we can remove even more code and make the whole thing more simple

File: packages/web-pkg/src/composables/actions/spaces/useSpaceActionsEditReadmeContent.ts [hunk 1]
```diff
@@ -3,20 +3,58 @@ import { computed } from 'vue'
...
+    try {
+      await clientService.webdav.getFileInfo(resources[0], { path: '.space' })
+    } catch (_) {
+      await createDefaultMetaFolder(resources[0])
+      markdownResource = await clientService.webdav.putFileContents(resources[0], {
+        path: '.space/readme.md'
+      })
+
+      const { data: updatedDriveData } = await clientService.graphAuthenticated.drives.updateDrive(
+        resources[0].id as string,
```
Comment (JammingBen) [inline packages/web-pkg/src/composables/actions/spaces/useSpaceActionsEditReadmeContent.ts:32]: I guess this comes from former times.

```suggestion
        resources[0].id,
```

File: packages/web-pkg/src/composables/actions/spaces/useSpaceActionsEditReadmeContent.ts [hunk 2]
```diff
@@ -3,20 +3,58 @@ import { computed } from 'vue'
...
+        {
+          special: [
+            {
+              specialFolder: {
+                name: 'readme'
+              },
+              id: markdownResource.id as string
```
Comment (JammingBen) [inline packages/web-pkg/src/composables/actions/spaces/useSpaceActionsEditReadmeContent.ts:39]: Same here.

```suggestion
              id: markdownResource.id
```

**Review by JammingBen — COMMENTED** [review]
2 nitpicks, rest LGTM 👍 

I noticed that the space images coming from the sever are a bit bigger than our icons, which can cause some flickering during/after loading. Maybe we could introduce a loading spinner in the future? Just an idea, out of scope here anyway.

**Comment by AlexAndBear** [comment]
👍  for the spinner, in general a good idea, since we can't predict the internet speed on the client

**Review by JammingBen — APPROVED** [review]

**Pull request merged**